src/pages/enterprise/auth-connect/azure-ad-b2c.md

```diff
@@ -0,0 +1,146 @@
+        redirectUri: 'FILL_IN',
+        // requested scopes from provider
+        scope: 'FILL_IN',
+        // the audience, if applicable
+        audience: 'FILL_IN',
+        // the URL to redirect to after log out
+        logoutUrl: 'FILL_IN',
+        // The type of iOS webview to use. 'shared' will use a webview that can
+        // share session/cookies on iOS to provide SSO across multiple apps but
+        // will cause a prompt for the user which asks them to confirm they want
+        // to share site data with the app. 'private' uses a webview which will
+        // not prompt the user but will not be able to share session/cookie data
+        // either for true SSO across multiple apps.
+        iosWebView: 'private',
+        // required if running on the Web
+        clientSecret: ''
+    };
+    
+    super(azureConfig);
+    }
+}
+```
+
+Some of these `IonicAuthOptions` values are unique, and must be set based on your Azure AD app’s details:
+
+* `platform`: Use “cordova” or “capacitor” accordingly.
+* `clientID`: Your app’s _Application ID_. Example: cebbb0be-d578-4bbd-9712-4b0fe05c06aa
+* `redirectUri`: The URI to redirect to after the user has logged in. Use the same AUTH_URL_SCHEME variable value (App Id) from when the Auth Connect plugin was installed. Example: com.company.app://callback
+* `logoutUrl`: The URI to redirect to after the user has logged out. Example: com.company.app://login?logout=true
+* `audience`: Your custom API URL of choice, such as `https://api.myapp.com`.
+* `scope`: Unlock access to protected resources, such as read/write permissions. `offline_access` is minimally required. Example: openid offline_access email picture profile
+
+The `discoveryUrl` can be found by navigating to User flows (policies) -> [Select User Flow] -> Overview tab -> Run user flow button. The discovery link is at the top page and will look like the following format:
+
+`https://B2C-TENANT-NAME.b2clogin.com/B2C-TENANT-NAME.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=POLICY-NAME`
+
+Where `B2C-TENANT-NAME` is your tenant name and the `POLICY-NAME` is the name of the User Flow created earlier.
+
+### What's Next?
+
+Check out the full list of [configuration options](/docs/enterprise/auth-connect#ionicauthoptions) available, then implement the [other steps](/docs/enterprise/auth-connect#workflow) in the Auth Connect workflow.
+
+## Implementing password reset using custom user flows/policies
+
+To implement password reset functionality, a custom User Flow needs to be created. Navigate to the `User flows (policies)` page, then click the "New user flow" button. Next, select the "Password reset" user flow type. As part of the `Application claims` section, choose "Email Addresses" at a minimum. After the user flow has been created, select it from the User Flow list, then click "Run user flow" from the Overview tab. Note the URL at the top of the page - use it as the discovery url for password reset.
```

Comment:
Moved this from the main Auth Connect page to here, and expanded on the details. including, adding a code snippet.